src/grammar.coffee

```diff
@@ -533,6 +534,34 @@ grammar =
     o '...',                                     -> new Expansion
   ]
 
+  # The **ArgElisionList** is the list of objects, contents of an array literal
+  # (i.e. comma-separated expressions and elisions). Newlines work as well.
+  ArgElisionList: [
+    o 'ArgElision'
+    o 'ArgElisionList , ArgElision',                                      -> $1.concat $3
+    o 'ArgElisionList TERMINATOR Arg',                                    -> $1.concat $3
+    o 'ArgElisionList , TERMINATOR ArgElision',                           -> $1.concat $4
+    o 'INDENT ArgElisionList OptComma OUTDENT',                           -> $2
+    o 'INDENT ArgElisionList , Elisions OUTDENT',                         -> $2.concat $4
+    o 'ArgElisionList OptComma INDENT ArgElisionList OptComma OUTDENT',   -> $1.concat $4
+    o 'ArgElisionList , Elisions INDENT ArgElisionList OptComma OUTDENT', -> $1.concat $3, $5
+    o 'ArgElisionList OptComma INDENT ArgElisionList , Elisions OUTDENT', -> $1.concat $4, $6
+  ]
+
+  ArgElision: [
+    o 'Arg',                  -> [$1]
+    o 'Elisions Arg',         -> [].concat $1, $2
+  ]
+
+  Elisions: [
```

Comment:
This should be called `ElisionList` for consistency with `ArgList`.

Reply:
Are you referring to the `Elisions`? They consist only of one or more `,` (`Elision`).
`ArgList` consists of  different elements: `Arg` (`Expression`, `Splat`, and `Expansion`) and ` ,`.

Or, perhaps you mean other elements: `ArgElision` or `ArgElisionList`?

Reply:
I was referring to `Elisions`, but you make a good point.